models/dex/dex_info.sql

```diff
@@ -90,6 +90,7 @@ FROM (VALUES
     , ('mauve', 'Mauve', 'Direct', 'mauve_org')
     , ('xchange', 'X7 Finance', 'Direct', 'X7_Finance')
     , ('mento', 'Mento', 'Direct', 'mento')
+    , ('hashflow', 'Hashflow', 'Direct', 'hashflow')
```

Comment:
Defined in https://github.com/duneanalytics/spellbook/blob/7f03125cbb8a2100bb4c924ddd14dd580702909c/models/dex/dex_info.sql#L35

```suggestion
```